tests/unit/xtriggers/test_workflow_state.py

```diff
@@ -80,7 +81,14 @@ def test_back_compat(tmp_run_dir):
     finally:
         conn.close()
 
+    # Test workflow_state function
     satisfied, _ = workflow_state(id_, task='mithril', point='2012')
     assert satisfied
     satisfied, _ = workflow_state(id_, task='arkenstone', point='2012')
     assert not satisfied
+
+    # Test back-compat (old suite_state function)
+    satisfied, _ = suite_state(id_, task='mithril', point='2012')
+    assert satisfied
+    satisfied, _ = suite_state(id_, task='arkenstone', point='2012')
```

Comment:
Worth testing the `suite` kwarg explicitly
```suggestion
    satisfied, _ = suite_state(suite=id_, task='mithril', point='2012')
    assert satisfied
    satisfied, _ = suite_state(suite=id_, task='arkenstone', point='2012')
```